runironic.sh

```diff
@@ -1,24 +1,24 @@
 #!/usr/bin/bash
 
 # Get environment settings and update ironic.conf
-IP=${IP:-"172.22.0.1"}
+PROVISIONING_INTERFACE=${PROVISIONING_INTERFACE:-"provisioning"}
+IRONIC_IP=`ip -4 address show dev $PROVISIONING_INTERFACE | grep -oP '(?<=inet\s)\d+(\.\d+){3}' | head -n 1`
```

Comment:
please use $() instead of backquotes to wrap the command
wrap $PROVISIONING_INTERFACE in double-quotes "